devices/ikea/starkvind_air_purifier.json

```diff
@@ -48,7 +61,16 @@
           "name": "attr/uniqueid"
         },
         {
-          "name": "config/on"
+          "name": "config/ledindication",
+          "parse": {"at": "0x0003", "cl": "0xfc7d", "ep": 1, "fn": "zcl", "mf": "0x117c", "eval": "Item.val = !Attr.val"},
+          "write": {"at": "0x0003", "cl": "0xfc7d", "dt": "0x10", "ep": 1, "fn": "zcl", "mf": "0x117c", "eval": "!Item.val"},
+          "default": true

```

Comment:
The default value could be removed, as it comes already with the resource item. I'd recommend to add the read function as well, otherwise the state changes has probably no chance to succeed. If you've configured reporting for this attribute, it's also advisable to add `"refresh.interval"` with a duration time of max reporting interval + 60 (to allow some polling fallback)